privaterelay/templates/settings.html

```diff
@@ -0,0 +1,121 @@
+{% extends "base.html" %}
+
+{% load socialaccount %}
+{% load static %}
+{% load relay_tags %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+{% block javascript %}
+  <script src="{% static 'js/settings.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+
+{% get_social_accounts request.user as accounts %}
+<firefox-private-relay-addon-data 
+  data-fxa-subscriptions-url="{{ settings.FXA_SUBSCRIPTIONS_URL }}" 
+  data-premium-prod-id="{{ settings.PREMIUM_PROD_ID }}" 
+  data-premium-price-id="{{ settings.PREMIUM_PRICE_ID }}" 
+  data-aliases-used-val="{{ relay_addresses|length }}" 
+  data-emails-forwarded-val="{{ user_profile.emails_forwarded }}"
+  data-emails-blocked-val="{{ user_profile.emails_blocked }}"
+  {% with request.user.profile_set.first as user_profile %}
+    data-premium-subdomain-set="{{ user_profile.subdomain }}"
+  {% endwith %}
+  data-premium-enabled="{{ settings.PREMIUM_ENABLED }}"
+>
+</firefox-private-relay-addon-data>
+
+<main id="settings-main" class=" bg-light dashboard-container " data-api-token="{{ user_profile.api_token }}">
+  <div class="c-meta">
+    <h1 class="mzp-u-title-md c-settings-headline">{% ftlmsg 'settings-headline' %}</h1>
+    <ul class="c-meta--links">
+      <li>
```

Comment:
This is still missing the "Contact us" link, because I didn't know the link target yet.

Reply:
This is for free users too, so I'm not sure we can link them to a "contact us"? @tcinotto - where do we want 200K+ free users to send their contact?